easybuild/easyconfigs/h/HTSeq/HTSeq-0.9.1-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = "PythonPackage"
+
+name = 'HTSeq'
+version = '0.9.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www-huber.embl.de/users/anders/HTSeq/'
+description = """A framework to process and analyze data from high-throughput sequencing (HTS) assays"""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+checksums = ['af5bba775e3fb45ed4cde64c691ebef36b0bf7a86efd35c884ad0734c27ad485']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('Pysam', '0.14', versionsuffix),
+    ('matplotlib', '2.1.0', versionsuffix),
+]
+
```

Comment:
As this is a PythonPackage you should add 
download_dep_fail = True
as Travis says.

The same goes for all easyconfigs here that are PythonPackage based.

Reply:
Thank you @akesandgren . I have added the option to all the four eb files.